tests/market/test_forward.py

```diff
@@ -122,3 +124,23 @@ def test_market_rotation(self, market_class, rotator_class, expected_market_coun
         # Market should be deleted if the rotation time has been reached
         rotator.rotate(rotation_time)
         count_orders_in_buffers(forward_markets, expected_market_count - 1)
+
+    @staticmethod
+    @patch("gsy_e.models.market.forward.ConstSettings.ForwardMarketSettings."
+           "ENABLE_FORWARD_MARKETS", True)
+    def test_clock_updates():
```

Comment:
The same comment as [above](https://github.com/gridsingularity/gsy-e/pull/1506/files#r943238217) applies here: which method are we testing? I would say that we're basically testing the `market.update_clock()` part of `Area.execute_actions_after_tick_event()`. One possible alternative is to create `test_execute_actions_after_tick_event()`, make sure it's correctly calling `market.update_clock()`, and then test `market.update_clock()` separately.